styles.css

```diff
@@ -1297,6 +1292,208 @@ button {
   top: 6290px;
 }
 
+.github-icon {
+  width: 20px;
+  height: 15px;
+  align-items: flex-end;
+}
+
+.popup-btn:hover {
+  background-color: rebeccapurple;
+  color: #fff;
+}
+
+.project-detail-image {
+  width: 100%;
+  height: 220px;
+}
+
+.modal {
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  height: 100%;
+
+  /* bottom: -200%; */
```

Comment:
Great job team 🤝

- [ ] You may also kindly remove all commented code from your style sheet as this is not regarded as a good practice.